spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/CloudEurekaTransportConfig.java

```diff
@@ -18,12 +18,14 @@
 
 import com.netflix.discovery.shared.transport.EurekaTransportConfig;
 
-import lombok.Data;
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
@spencergibb, sure makes sense.  I have initially started to use commons-lang builders for `toString`, `equals` and `hashCode` methods on `@ConfigurationProperties` annotated classes in order to keep maintenance low.  I will make the requested changes, but wanted to ask whether there is a need to fully implemented `hashCode` and `equals` in `@ConfigurationProperties` classes?